index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    
+    
+</head>
+<body>
+  <header>
+    <nav>
+      <div class="topnav">
+
+    <a class = "logo">Grace</a>
+
+       <nav class ='hambuger'>
+        <nav class ="hambuger-bar"></nav>
+        <nav class ="hambuger-bar"></nav>
+        <nav class ="hambuger-bar"></nav>
```

Comment:
- [ ] As I can notice, you did not get very well what the comment suggested by the previous reviewer on this section. what they try to mean is that your hamburger menu is the one that is supposed to be in a navigation bar. Like so:
```html
<header>
   <div class="topnave">
       Logo
   </div>
  <nav>
    hamburger // you can use div to grab each one of the lines of the hamburger
  </nav>
</header>
```